app/src/main/java/com/example/journeyjournal/fragments/FeedFragment.java

```diff
@@ -147,11 +146,11 @@ protected void startLocationUpdates() {
         // Create the location request to start receiving updates
         LocationRequest mLocationRequest = LocationRequest.create();
         mLocationRequest.setPriority(LocationRequest.PRIORITY_HIGH_ACCURACY);
-        /* 10 secs */
-        long UPDATE_INTERVAL = 10 * 1000;
-        mLocationRequest.setInterval(UPDATE_INTERVAL);
-        /* 2 sec */
-        long FASTEST_INTERVAL = 2000;
+        /* 2 Mile */
+        long MIN_DISTANCE = 2 * 1609;
```

Comment:
move defined constant at beginning of class.

Reply:
resolved